packages/twitch-pubsub-client/src/Messages/PubSubSubscriptionMessage.ts

```diff
@@ -134,14 +149,14 @@ export class PubSubSubscriptionMessage {
 	 * Whether the subscription is a gift.
 	 */
 	get isGift(): boolean {
-		return this._data.context === 'subgift';
+		return this._data.context === 'subgift' || this._data.context === 'resubgift';
```

Comment:
The anon variants are missing here too, can you please add them?

Reply:
Done!